tests/test_cli.py

```
@@ -2477,6 +2477,124 @@ def test_start_codex_json(
+        mock_build_auto_config: MagicMock,
+        mock_apply: MagicMock,
+        _mock_is_published: MagicMock,
+        capsys: pytest.CaptureFixture,
+    ) -> None:
+        """When no Alpine row is published yet, ``--os alpine`` must thread
+        the OS through ``_build_auto_config`` (install-at-boot path) and
+        echo the flag value back in the JSON envelope."""
+        from smolvm.types import GuestOS
+
+        config = MagicMock(vm_id="sbx-claude")
+        mock_build_auto_config.return_value = (config, "/tmp/id_ed25519")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/id_ed25519"

(S108)